modules/ROOT/pages/_partials/data-sync/client-using-a-data-sync-client-to-query-a-mutation-on-a-data-sync-server.adoc

```diff
@@ -0,0 +1,101 @@
+[id="client-using-a-data-sync-client-to-query-a-mutation-on-a-data-sync-server-{context}"]
+= Using a Data Sync Client to query a mutation on a Data Sync Server
+
+Build a Data Sync Server and query the mutation.
+The Data Sync Server will be running locally in this procedure.
+
+.Prerequisites
+
+* The example in link:https://github.com/aerogear/mobile-docs/blob/master/modules/ROOT/pages/_partials/data-sync/server-create-a-graphql-schema-for-an-existing-data-source.adoc[Getting started with an example data source schema] runs locally.
+* Basic knowledge of `npm`.
+
+.Procedure
+
+. Create a basic link:https://webpack.js.org/guides/getting-started/[_webpack project_] repo.
```

Comment:
Would it be better to have the direct instructions, for creating the repo, in this doc instead of linking to the webpack page?  User may be a little confused about what exactly they will have to do on the webpack page, and there is always the chance the text on the webpack page could change.

Reply:
I do agree with "Would it be better to have the direct instructions, for creating the repo?" comment @briandooley